views/sidebar.ejs

```diff
@@ -0,0 +1,22 @@
+<small> You'll receive your email subscriptions at your primary email <b><%emails[0]%></b> by default.</small>
+<hr style="height:20px;visibility:hidden">
```

Comment:
Could you remove these `hr`s and wrap each block of text in a `<p>`? Right now I feel like there's a little too much space between each block, and this would be a cleaner solution as well.